package.json

```diff
@@ -297,6 +297,15 @@
       "label": "Automatically close window on start",
       "default": true
     },
+    {
+      "name": "customTimerFormBypass",
+      "label": "Custom Timer Form Bypass",
+      "title": "Custom Timer Form Bypass",
+      "type": "checkbox",
+      "required": false,
+      "description": "For 'Start Custom Timer', immediately start an untitled timer with default sound if any of the inline search bar hour, minute, or second fields are populated.",
+      "default": false
+    },
```

Comment:
"Skip Form if Arguments Given" might make for a more descriptive `label`, similarly instead of "inline search ... fields" in the description, using "arguments" (the standard Raycast terminology) might be clearer for users.

Also, add your Raycast handle to the `contributors` field in this file so you can be properly credited for your contribution :)